### PR TITLE
add CI

### PR DESCRIPTION
**Pull request by alexandercbooth**
Kicking the configuration here 🎉 
- This just tries to clone, install desktop and run it on CI.
- also adds spectron for later use

### CODE REVIEW

**Comment by alexandercbooth** [comment]
Note, I have already enabled this repo on circle 😄

File: index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// A simple test to verify a visible window is opened with no title
+var Application = require("spectron").Application;
+var assert = require("assert");
+var path = require("path");
+
+var appPath = "nteract/applications/desktop/dist/squashfs-root";
```
[the remark this answers is not in the document]
Comment (alexandercbooth) [reply index.js]: I think that is a great idea! Once we get a basic linux setup working, we can definitely do that 👍

**Comment by alexandercbooth** [comment]
>Feel free to merge if I don't when CI passes.

Awesome, will do. Probably will take a few tries to get it passing 😅

**Comment by alexandercbooth** [comment]
Passing, but it looks like `spectron` is timing out. I think we might have to do more custom setup for chromium. Will investigate tomorrow 🛌 😄

**Comment by rgbkrk** [comment]
Sounds good to me, might as well keep moving this on forward. 😄

**Comment by alexandercbooth** [comment]
OK great, thanks!

**Pull request merged**